### PR TITLE
Migrate to python 3.10

### PR DESCRIPTION
**Pull request by arturo-seijas**
Migrate python to version 3.10, supported by the newly release Indico 3.2.

### CODE REVIEW

File: indico-nginx.Dockerfile [hunk 1]
```diff
@@ -2,22 +2,18 @@ FROM ubuntu:jammy
...
+    && apt install -y libpq-dev python3-pip \
+    && python3 -m pip install --prefer-binary indico indico-plugins
 
 FROM ubuntu:jammy
 
 RUN apt update \
     && apt install -y nginx
 
-COPY --from=0 /usr/local/lib/python3.9/dist-packages/indico/web/static /srv/indico/static
+RUN ls /usr/local/lib/
```
Comment (mthaddon) [inline indico-nginx.Dockerfile:14]: Is this for debugging? Not quite sure why it's needed

Comment (arturo-seijas) [reply indico-nginx.Dockerfile:14]: Well spotted. Thx

**Review by mthaddon — APPROVED** [review]
LGTM, great to be able to switch to python3.10 natively.

**Comment by mthaddon** [comment]
This has been approved, but we're planning to hold off on landing it until we know the outcome of https://github.com/canonical/indico-operator/pull/85.

**Review by mthaddon — APPROVED** [review]
Adding another approval so this can be merged given we've established 3.2 works on staging

**Pull request merged**